makefile

```diff
@@ -95,4 +95,8 @@ docker_push_release:
 .PHONY: docker_push_master
 docker_push_master:
 	@docker login -u $(DOCKERHUB_USERNAME) -p $(DOCKERHUB_PASSWORD)
-	docker push $(DOCKER_MASTER_TAG)
\ No newline at end of file
+	docker push $(DOCKER_MASTER_TAG)
+
+.PHONY: generate
+generate:
+	go generate ./cmd/metrics-api/metrics-api.go
```

Comment:
this failed for me when checking out the branch due to the `cors` dep not having been installed, maybe make this dependant on `make setup`

Reply:
go-swagger is not included in the dep dependencies. dep only allows deps that are actually being used in code. The Readme has the appropriate instructions to ensure this issue doesn't happen :)

Reply:
yes, I meant cors just because I was coming from having a vendor dir with all but the new dependency in it.

I know the readme points to `make setup` but we already have `make build|build_linux` depend on it so `make generate` could too.

And yeah, I'd like golang to have something like `devDependencies` for binaries.